extensions/amp-story/1.0/bookend/components/article.js

```diff
@@ -79,16 +91,23 @@ export class ArticleComponent {
    * @return {!Element}
    * @override
    * */
-  buildTemplate(articleData, doc) {
+  buildElement(articleData, doc) {
     const html = htmlFor(doc);
     //TODO(#14657, #14658): Binaries resulting from htmlFor are bloated.
-    const template =
+    const el =
         html`
         <a class="i-amphtml-story-bookend-article
           i-amphtml-story-bookend-component"
           target="_top">
+          <h2 class="i-amphtml-story-bookend-article-heading" ref="heading">
+          </h2>
+          <div class="i-amphtml-story-bookend-component-meta" ref="meta"></div>
         </a>`;
-    addAttributesToElement(template, dict({'href': articleData.url}));
+    addAttributesToElement(el, dict({'href': articleData.url}));
```

Comment:
Nit: here and in all this PR, unless you're doing it to avoid minification issues or something I'm not aware of, please use ``{href: 'foo'}`` over ``{'href': 'foo'}``

Reply:
When using dict, we must pass a string key, otherwise the linter complains.